groups/views.py

```diff
@@ -184,13 +172,41 @@ def email_subscribers(self, comment):
                 to=user.email,
                 subject=NEW_COMMENT_SUBJECT.format(discussion=comment.discussion.name),
                 template_name='groups/emails/new_comment.txt',
+                reply_to=get_reply_address(comment.discussion, user, self.request),
                 context={
                     'comment': comment,
                     'user': user,
-                }
+                },
             )
 
 
+class CommentPostView(CommentEmailMixin, CreateView):
+    """
+    Base class for views that post a comment to a particular discussion.
+
+    Must be initialised with form_class and template_name attributes, as required by
+    the superclass, CreateView.
+    """
+    model = models.BaseComment
+
+    def dispatch(self, request, *args, **kwargs):
+        pk = self.kwargs['pk']
+        self.discussion = models.Discussion.objects.select_related('group').get(pk=pk)
```

Comment:
Discussions cannot yet be deleted... even when they can, they'll probably get an "archived" flag set.